pyproject.toml

```diff
@@ -21,17 +21,20 @@ classifiers = [
 [tool.poetry.dependencies]
 python = ">=3.7.1, <3.11"
 pandas = "^1.3.4"
-arrow = "^1.2.0"
```

Comment:
are we sure about this one?

Reply:
`arrow` on PyPI seems to be an unrelated library (to actual Apache Arrow) that does date processing, https://pypi.org/project/arrow/. I grepped the codebase, and seems like we don't use it at all. The PyArrow dependency is still here by the way (just right under that line)